TypewiseAlert.Test/TypewiseAlertTest.cs

```diff
@@ -3,13 +3,43 @@
 
 namespace TypewiseAlert.Test
 {
-  public class TypewiseAlertTest
-  {
-    [Fact]
-    public void InfersBreachAsPerLimits()
+    public class TypewiseAlertTest
```

Comment:
test cases for email ,controller and console notification not found

Reply:
i missed the check in of TypeWiseAlerttest checkin. I have convered the test cases.